### PR TITLE
LPD-1720 Disable repeatable buttons when onBlur is triggered

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPD-1720

### CODE REVIEW

**Review by cgmonte — COMMENTED** [review]
Hey @danvb112, nice and simple take on solving this long-standing issue in forms. I have tested and it works. Please, check my comments, as I think you may have forgotten to detach the event listener and made an unnecessary change.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js [hunk 1]
```diff
@@ -284,6 +287,16 @@ export function FieldBase({
...
 
+	useEffect(() => {
+		Liferay.on('disableRepeatableButton', () => {
+			setDisableRepeatableButton(true);
+
+			setTimeout(() => {
+				setDisableRepeatableButton(false);
+			}, 1000);
+		});
+	}, []);
```
[the remark this answers is not in the document]
Comment (danvb112) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js:310]: Hey @cgmonte, thanks for noticing that, yes, I completely forgot to detach on component unmount! :smile:

**Comment by danvb112** [comment]
ci:test:sf

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by cgmonte** [comment]
ci:test:forms

**Comment by danvb112** [comment]
ci:test:data-engine

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
Analyzing ci results...

**Comment by danvb112** [comment]
Failures in tests results are not related

**Comment by danvb112** [comment]
ci:forward:force

**Comment by danvb112** [comment]
ci:forward:force

**Pull request closed without merging**